gas-measurement.sh

```diff
@@ -6,7 +6,7 @@ default_json_file="${out_dir}/gas-measurement.json"
 function run_gas_measurement() {
   local json_file=$1
   local tmp_file="$1.tmp"
-  forge test --mp "test/core/gas/*" -vv --silent --json > "${json_file}"
+  forge test --mp "$GAS_MATCH_PATH" -vv --silent --json > "${json_file}"
```

Comment:
Would be nice if `GAS_MATCH_PATH` was a parameter instead and/or had a default value.

What happens if I run `gas-measurement.sh` without this env var set? That's what I normally do and what the instructions in the README.md say :-)

Reply:
ok 😔